scss/components/notification.scss

```diff
@@ -0,0 +1,311 @@
+@import "./../mixins";
+@import "./../functions";
+
+$block: #{$fd-namespace}-notification;
+.#{$block} {
+    // variables
+    $fd-notification-max-width: 60rem !default;
+    $fd-notification-max-width-m: 640px !default;
+    $fd-notification-min-width-m: 320px !default;
+    $fd-notification-width-s: 320px !default;
+
+    $fd-notification-background-color: var(--sapList_Background, $sapList_Background) !default;
+    $fd-notification-darken-background-color: $sapList_Background !default;
+
+    $fd-notification-box-shadow: var(--sapContent_Shadow0, $sapContent_Shadow0) !default;
+
+    $fd-notification-border-radius: 0.25rem !default;
+    $fd-notification-group-border-radius-first-child: 0.25rem 0.25rem 0 0 !default;
+    $fd-notification-group-border-radius-last-child: 0 0 0.25rem 0.25rem !default;
+
+    $fd-notification-indicator-color-error: var(--sapErrorColor, $sapErrorColor) !default;
+    $fd-notification-indicator-color-warning: var(--sapWarningColor, $sapWarningColor) !default;
+    $fd-notification-indicator-color-success: var(--sapSuccessColor, $sapSuccessColor) !default;
+    $fd-notification-indicator-color-information: var(--sapInformationColor, $sapInformationColor) !default;
+
+    $fd-notification-indicator-border-color-error: var(--sapErrorBorderColor, $sapErrorBorderColor) !default;
+    $fd-notification-indicator-border-color-warning: var(--sapWarningBorderColor, $sapWarningBorderColor) !default;
+    $fd-notification-indicator-border-color-success: var(--sapSuccessBorderColor, $sapSuccessBorderColor) !default;
+    $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor, $sapInformationBorderColor) !default;
+
+    $fd-notification-title-color: var(--sapGroup_TitleTextColor, $sapGroup_TitleTextColor) !default;
+    $fd-notification-content-text-color: var(--sapContent_LabelColor, $sapContent_LabelColor) !default;
+
+    // mixins
+    @mixin fd-priority-indicator($type) {
+        margin-top: 1px;
+        padding: 1rem 0.5rem 0 1rem;
+        @include fd-icon("message-#{$type}", "m");
+    }
+
+
+    // ------------------------ CONTAINER ------------------------
+    @include fd-reset();
+    max-width: $fd-notification-max-width;
+    background-color: $fd-notification-background-color;
+    box-shadow: $fd-notification-box-shadow;
+    border-radius: $fd-notification-border-radius;
+    border-left: 0.375rem solid;
+    border-left-color: var(--sapNeutralBorderColor, $sapNeutralBorderColor);
+    margin-bottom: 0.5rem;
+
+    @include fd-rtl() {
+        border-left: none;
+        border-right: 0.375rem solid;
+        border-right-color: var(--sapNeutralBorderColor, $sapNeutralBorderColor);
+    }
+
+
+    // ------------------------ HEADER ------------------------
+    &__header {
+        @include fd-reset();
+        display: flex;
+        box-shadow: inset 0 -0.0625rem var(--sapPageHeader_BorderColor, $sapPageHeader_BorderColor);
+    }
+
+    &__indicator {
```

Comment:
I think we need some sizing resets here:
![Screen Shot 2019-09-11 at 3 14 23 PM](https://user-images.githubusercontent.com/29607818/64739462-00d38580-d4a7-11e9-99d0-cfe6ebec86e8.png)

Reply:
done